spec/acceptance/self_service_spec.rb

```diff
@@ -149,6 +149,17 @@ class {'self_service':
         expect(result.stdout).to match(%r{false})
         run_shell('puppet config set use_cached_catalog false', expect_failures: false)
       end
+      it 'if S0032 conditions for false are met' do
+        configdir = if Facter.value(:os)['family'].eql?('RedHat') || Facter.value(:os)['family'].eql?('Suse')
+                      '/etc/sysconfig'
+                    else
+                      '/etc/default'
+                    end
+        run_shell("touch #{configdir}/mcollective")
```

Comment:
I think it should be run_shell("touch ${configdir}/mcollective") here and the removal below to fix the tests